src/D2L.Bmx/WindowsParentProcess.cs

```diff
@@ -0,0 +1,46 @@
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace D2L.Bmx;
+
+internal partial class WindowsParentProcess {
```

Comment:
would be nice to add a comment noting the PowerShell implementation that's referenced
https://github.com/PowerShell/PowerShell/blob/26f621952910e33840efb0c539fbef1e2a467a0d/src/System.Management.Automation/engine/ProcessCodeMethods.cs